tests/plugins/mocha.test.support/hooks/hook.js

```diff
@@ -40,6 +40,25 @@ exports.init = (logger, config, cli) => {
 		}
 		done();
 	});
+
+	cli.on('build.post.install', async (builder, done) => {
+		try {
+			if (builder.platformName === 'android') {
+				// Grand all needed permissions on the Android device.
+				await grantAndroidPermissions(logger, builder);
+
+			} else if ((builder.platformName === 'iphone') && (cli.argv.target === 'simulator')) {
+				// Grant all needed permissions on the iOS simulator.
+				await xcrun([ 'simctl', 'privacy', builder.simHandle.udid, 'grant', 'all', builder.tiapp.id ]);
+
+				// Re-launch app in case granting permissions forced-quit it.
+				await xcrun([ 'simctl', 'launch', builder.simHandle.udid, builder.tiapp.id ]);
```

Comment:
I assume this is because of the note above where we're not actually firing post-install, pre-launch - we're firing post-launch?

Reply:
Exactly right. We'd be able to remove this line if we could fire the post install event before the launch.